framework/src/postprocessors/MemoryUsage.C

```diff
@@ -44,6 +44,8 @@ MemoryUsage::MemoryUsage(const InputParameters & parameters)
   : GeneralPostprocessor(parameters),
     _mem_type(getParam<MooseEnum>("mem_type").getEnum<MemType>()),
     _value_type(getParam<MooseEnum>("value_type").getEnum<ValueType>()),
+    _value(0.0),
+    _peak_value(0.0),
```

Comment:
Oh man! I even noticed that you didn't initialize these in my review but I figured it was ok because you set them to zero in `initialize`. I guess I should have gone with my gut...